homework-1/TestStudentRepository.java

```diff
@@ -0,0 +1,39 @@
+package com.hw.student;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.time.LocalDate;
+
+
+/**
+ * Class that tests the StudentRepository
+ */
+public class TestStudentRepository {
+
+    @Test(expected = NullPointerException.class)
```

Comment:
in addStudent ai conditia  if (student == null)  si sa arunca o exceptie custom. Atunci nu ar trebui la expected sa fie NullPointerException in acest caz